object-search/pods/craft.yml

```diff
@@ -0,0 +1,56 @@
+!CompoundExecutor
+components:
+  - !ImageReader
+    with:
+      from_bytes: True
+    metas:
+      name: img_read
+  - !ImageNormalizer
```

Comment:
Just a note for our improvement, in the `TorchObjectDetectionSegmenter` hub tests, no normalization was applied before, it does not matter but it may lead to a little missuse, because I think since the `backbone` is as any other classification model, it should benefit from this normalization.